packages/deployments/contracts/contracts/core/connext/helpers/RelayerProxy.sol

```diff
@@ -46,12 +46,8 @@ contract RelayerProxy is ProposedOwnable, ReentrancyGuard, GelatoRelayFeeCollect
   ISpokeConnector public spokeConnector;
   // special consideration for Autonolas keeper
   address public autonolas;
```

Comment:
do we want to keep this when everything else is removed?

Reply:
if we are keeping autonolas on the `RelayerProxy` so it is usable on spokes, wouldn't we want to keep all of the helpers in the same contract? im definitely missing some context on what this is and why its being moved around right now